app/models/user.rb

```diff
@@ -25,4 +25,15 @@ def cards_for_review
   def has_authentications?
     !email.blank? && authentications.present?
   end
+
+  def review_notification
+    User.all.each do |user|
+      unless user.cards_for_review.empty?
+        CardsMailer
+            .pending_cards_notification(user, user.cards_for_review.length)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.